packages/react-search-ui-views/src/SearchBox.js

```diff
@@ -15,32 +25,48 @@ function SearchBox(props) {
     isFocused,
     inputProps,
     onChange,
-    onSelectAutocomplete,
     onSubmit,
     value
   } = props;
   const focusedClass = isFocused ? "focus" : "";
 
-  return (
-    <form onSubmit={onSubmit}>
-      <Downshift
-        inputValue={value}
-        onChange={onSelectAutocomplete}
-        onInputValueChange={onChange}
-        itemToString={item =>
-          // TODO
-          item
-            ? item[autocompleteResults.titleField]
-              ? item[autocompleteResults.titleField].raw ||
-                item[autocompleteResults.titleField].snippet
-              : item.suggestion
-            : ""
+  const onSelectAutocomplete =
+    props.onSelectAutocomplete ||
```

Comment:
You could omit `props.` and add `onSelectAutocomplete` to the destructured props above

Reply:
Good suggestion. I did it this was because I am basically trying to default the behavior. I couldn't find a good way to do that with destructuring.